### PR TITLE
Initial setup

### PR DESCRIPTION
**Pull request by aslakhol**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    "eject": "react-scripts eject",
+    "storybook": "start-storybook -p 9009 -s public",
+    "build-storybook": "build-storybook -s public",
+    "lint": "tsc --noEmit && eslint '*/**/*.{js,ts,tsx}' --quiet --fix"
+  },
+  "eslintConfig": {
+    "extends": "react-app"
+  },
+  "browserslist": {
+    "production": [
```
Comment (hernil) [inline package.json]: You can be a bit more restrictive here, given your audience, (no IE?) and save yourselves a good amount of polyfills in the payload you need to ship.

File: src/react-app-env.d.ts [hunk 2]
```diff
@@ -0,0 +1 @@
+/// <reference types="react-scripts" />
```
Comment (hernil) [inline src/react-app-env.d.ts]: Do we need this file?

File: src/index.tsx [hunk 3]
```diff
@@ -0,0 +1,5 @@
+import React from 'react';
```
Comment (hernil) [inline src/index.tsx]: This file (probably as a ts not tsx file) will be the entrypoint of the apps using the component library thus this file should contain exports of the modules you want to expose to the world.

File: src/react-app-env.d.ts (hunk 2 above)
Comment (aslakhol) [reply src/react-app-env.d.ts]: Yes

File: src/index.tsx (hunk 3 above)
Comment (aslakhol) [reply src/index.tsx]: Yes, we will do this when there are modules to export.

Comment (hernil) [reply src/index.tsx]: :+1:

File: src/react-app-env.d.ts (hunk 2 above)
Comment (hernil) [reply src/react-app-env.d.ts]: Then this file should probably contain more than a commented line? :-)

**Pull request merged**